azure-devops/projects/io-backend-projects/service_connections.tf

```diff
@@ -21,3 +21,31 @@ resource "azuredevops_serviceendpoint_azurerm" "DEV-IO" {
   azurerm_spn_tenantid      = data.azurerm_key_vault_secret.key_vault_secret["TTDIO-SPN-TENANTID"].value
   azurerm_subscription_id   = data.azurerm_key_vault_secret.key_vault_secret["TTDIO-DEV-IO-SUBSCRIPTION-ID"].value
 }
+
+# Github service connection (read-only)
+resource "azuredevops_serviceendpoint_github" "pagopa-github-bot-ro" {
+  depends_on = [azuredevops_project.project]
+
+  project_id            = azuredevops_project.project.id
+  service_endpoint_name = "pagopa-github-bot-ro"
+  auth_personal {
+    personal_access_token = data.azurerm_key_vault_secret.key_vault_secret["pagopa-github-bot-ro-TOKEN"].value
```

Comment:
What if we name them by their purpose instead of by their account? We would be free to change account without being forced to rename those keys. And, we can use the same name for the token on the github account itself.

Proposals:
* `io-devops-ro-TOKEN`
* `azure-devops-ro-TOKEN`
* `io-azure-devops-ro-TOKEN`

Reply:
Good idea, no problem. Also add `github` in name

- `azure-devops-github-ro-TOKEN`
- `azure-devops-github-rw-TOKEN`
- `azure-devops-github-pr-TOKEN`

Reply:
do we plan to use the same token outside IO? If not, I'd add `io-` prefix

Reply:
I don't have an answer :) but this repo is called io-gitops, so it makes sense to add `io-` prefix